apps/web/pages/v2/apps/[slug]/index.tsx

```diff
@@ -0,0 +1,105 @@
+import fs from "fs";
+import matter from "gray-matter";
+import { GetStaticPaths, GetStaticPropsContext } from "next";
+import { MDXRemote } from "next-mdx-remote";
+import { serialize } from "next-mdx-remote/serialize";
+import Image from "next/image";
+import Link from "next/link";
+import path from "path";
+
+import { getAppWithMetadata } from "@calcom/app-store/_appRegistry";
+import prisma from "@calcom/prisma";
+
+import { inferSSRProps } from "@lib/types/inferSSRProps";
+
+import App from "@components/v2/apps/App";
+
+const components = {
+  a: ({ href = "", ...otherProps }: JSX.IntrinsicElements["a"]) => (
+    <Link href={href}>
+      <a {...otherProps} />
+    </Link>
+  ),
+  img: ({ src = "", alt = "", placeholder, ...rest }: JSX.IntrinsicElements["img"]) => (
+    <Image src={src} alt={alt} {...rest} />
+  ),
+  // @TODO: In v2 the slider isn't shown anymore. However, to ensure the v1 pages keep
+  // working, this component is still rendered in the MDX content. To skip them in the v2
+  // content we have to render null here. In v2 the gallery is shown by directly
+  // using the `items` property from the MDX's meta data.
```

Comment:
Because in v1 the `<Slider>` is part of the content (there can be text above and underneath the content), we can not remove it yet. Therefore we render an empty fragment here (null isn't allowed in MDXRemote's type 🤷‍♂️). As soon as v1 is gone, we can remove the `Slider` component from the MDX. In v2 we directly use the items from the MDX's meta data. (see line 50).

Reply:
I think we should be able to work out all the changes we need for the release and not hold back on these changes introducing workarounds, but to make it work the changes should not be picked up for production, or we simply expose App Pages v2 to the public earlier. Not sure. @zomars?

Reply:
not worth the effort for now